robottelo/ui/contenthost.py

```diff
@@ -112,6 +112,21 @@ def validate_subscription_status(self, name, expected_value=True,
                 return True
         return False
 
+    def get_subscription_color_and_status(self, name):
+        """Return content host subscription status color and name
+
+        :returns a tuple (color, status_name) or None
+        """
+        self.search(name)
+        sub_element = self.wait_until_element(
+            locators['contenthost.subscription_status'] % name)
+        if sub_element:
+            sub_class_attributes = sub_element.get_attribute('class').split()
+            color = sub_class_attributes[0]
+            status = sub_class_attributes[-1].split('-')[-1]
```

Comment:
nitpick: a 1-linear comment with example output would be very helpful to understand how are you splitting the string and why